src/glue/lib/targets/s3-target.ts

```diff
@@ -20,11 +37,26 @@ export class S3Target implements ICrawlerTarget {
   private readonly _exclusions: string[] = [];
 
   // Input properties
+  /**
+   * Bucket to use as the Target
+   */
   public readonly bucket: IBucket;
-  public readonly connection?: Connection;
-  public readonly exclusions?: string[];
-  public readonly keyPrefix?: string;
-  public readonly sampleSize?: string;
+  /**
```

Comment:
Indentation looks off.

Reply:
Tabs vs Spaces issue. Should be fixed now